lib/zora/filterCasts.ts

```diff
@@ -0,0 +1,47 @@
+import { Cast } from '@neynar/nodejs-sdk/build/neynar-api/v2';
+import getCastsMetadataLink from './getCastsMetadataLink';
+import getCastContractMapping from './mapCastToContract';
+
+/**
+ * Filter zora feed/casts by the audio mimeType
+ * 
+ */
+async function filterZoraFeed(casts: Cast[]) {
+  try {
+    const data = await getCastsMetadataLink(getCastContractMapping(casts));
+
+    const castsMetaData = await Promise.all(data.map(cast => (
+      fetch(`https://gateway.pinata.cloud/${cast.ipfs}`, { cache: 'force-cache' })
```

Comment:
for IPFS gateway can you please use `cloudflare-ipfs.io`
- ex: `https://cloudflare-ipfs.com/ipfs/{HASH}`